OutOfSchool/OutOfSchool.WebApi/Services/IWorkshopServicesCombiner.cs

```diff
@@ -29,6 +29,14 @@ public interface IWorkshopServicesCombiner
     /// The task result contains a <see cref="List{ShortEntityDto}"/> that contains elements from the input sequence.</returns>
     Task<List<ShortEntityDto>> GetWorkshopListByProviderId(Guid providerId);
 
+    /// <summary>
+    /// Get all workshop cards (Id, Title) with the specified provider's Id.
+    /// </summary>
+    /// <param name="providerId">Provider's key.</param>
+    /// <returns>A <see cref="Task{ShortEntityDto}"/> representing the result of the asynchronous operation.
+    /// The task result contains a <see cref="List{ShortEntityDto}"/> that contains elements from the input sequence.</returns>
+    Task<List<ShortEntityDto>> GetWorkshopListByProviderAdminId(Guid providerId);
```

Comment:
In case this method will return other type of collection should we also rename it ?
Meaning `GetWorkshopsByProviderAdminId` sounds better as for me.